app/views/dashboards/show.html.erb

```diff
@@ -26,7 +26,14 @@
       <button class="btn btn-default" ng-click="decreaseEndTime()" title="Decrease end time">
         <i class="icon-fb"></i>
       </button>
-      <input type="text" class="form-control" datetime-picker datetime="globalConfig.endTime" data-format="yyyy-MM-dd" placeholder="Until">
+      <span class="until_container">
+        <input type="text" class="form-control" datetime-picker datetime="globalConfig.endTime" data-format="yyyy-MM-dd" placeholder="Until">
+        <span class="until_clear global pointer"
```

Comment:
Doubling the bonus points.

On Fri, Jan 23, 2015 at 8:01 PM, juliusv <notifications@github.com> wrote:

> In app/views/dashboards/show.html.erb
> <https://github.com/prometheus/promdash/pull/282#discussion_r23491653>:
>
> > @@ -26,7 +26,14 @@
> >        <button class="btn btn-default" ng-click="decreaseEndTime()" title="Decrease end time">
> >          <i class="icon-fb"></i>
> >        </button>
> > -      <input type="text" class="form-control" datetime-picker datetime="globalConfig.endTime" data-format="yyyy-MM-dd" placeholder="Until">
> > +      <span class="until_container">
> > +        <input type="text" class="form-control" datetime-picker datetime="globalConfig.endTime" data-format="yyyy-MM-dd" placeholder="Until">
> > +        <span class="until_clear global pointer"
>
> The "bonus points" seem quite easy though: simply add this to the span:
>
> ng-show="globalConfig.endTime"
>
> (same in the widget_time_options_template.html)
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/promdash/pull/282/files#r23491653>.
>